integration_tests/snaps/asset-tracking/snapcraft.yaml

```diff
@@ -10,3 +10,39 @@ parts:
     plugin: nil
     stage-packages: ['hello=2.10-1']
     build-packages: ['hello=2.10-1']
+  git-part:
+    plugin: nil
+    source: git-source
+    source-type: git
+  git-part-branch:
+    plugin: nil
+    source: git-source
+    source-type: git
+    source-branch: feature
```

Comment:
I personally would find this more clear if the branch was called test-branch, or something like that.
I get confused making the link feature<->branch.